app/controllers/accounts/connected_accounts_controller.rb

```diff
@@ -8,6 +8,7 @@ class ConnectedAccountsController < ApplicationController
     layout 'account_side_nav'
 
     def show
+      analytics.track_event(:connected_accounts_page_visited)
```

Comment:
We shouldn't be calling `track_event` directly, we should call `analytics.connected_accounts_page_visited`.

```suggestion
      analytics.connected_accounts_page_visited
```